app/controllers/api/v0/forem_directories_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V0
+    class ForemDirectoriesController < ApiController
+      def index
+        render json: {
+          cover_image_url: Settings::General.main_social_image,
+          description: Settings::Community.community_description,
+          logo_image_url: Settings::General.logo_png,
+          name: Settings::Community.community_name,
+          tagline: Settings::Community.tagline,
+          version: "edge.#{Time.current.strftime('%Y%m%d')}.0",
+          visibility: visibility
+        }.to_json,
```

Comment:
_nonblocking:_ `render json: value` will automatically call `.to_json` (actually, it's probably more like `ActiveSupport::JSON.generate` or something) on `value`, so an explicit `.to_json` here isn't needed. It's kinda cool because it lets you pass serializer instances directly to it.

```suggestion
        },
```